tests/data/test_LightlyDataset.py

```diff
@@ -339,3 +339,19 @@ def test_transform_setter(self, dataset: LightlyDataset = None):
     def test_no_dir_no_transform_fails(self):
         with self.assertRaises(ValueError):
             LightlyDataset(None, transform=torchvision.transforms.ToTensor())
+
+    @unittest.skipUnless(VIDEO_DATASET_AVAILABLE, "PyAV or CV2 is/are not installed")
+    def test_dataset_get_filenames(self):
+        self.create_video_dataset()
+        dataset = LightlyDataset(input_dir=self.input_dir)
+        
+        # get filenames using VideoDataset.get_filenames
+        video_dataset_filenames = dataset.dataset.get_filenames()
+        
+        # get filenames using calls to VideoDataset.get_filename(index)
+        get_filenames = VideoDataset.get_filenames
+        del VideoDataset.get_filenames
+        lightly_dataset_filenames = dataset.get_filenames()
+        VideoDataset.get_filenames = get_filenames
```

Comment:
This is a great way to test it, good idea!

I think the code would be easier understandable with
`lightly_dataset_filenames = super(VideoDataset, dataset).get_filenames()`.

Reply:
I don't think this call to super works. But I made the call more explicit.

Reply:
Thanks, that should work as well.